app/uk/gov/hmrc/apinotificationqueue/controller/CustomHeaderNames.scala

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
I think there is missing DEBUG level logging in `ApiSubscriptionFieldResponse` for URL etc